konvoy: Update docs for konvoy release v1.5.0-beta.8 (#3020)
mesosphere/dcos-docs-site: pages/ksphere/konvoy/1.5.0-beta/autoscaling/index.md

<p class="message--note"><strong>NOTE: </strong> This feature is only supported for AWS and Azure cloud providers.</p>

> <p class="message--note"><strong>IMPORTANT: </strong> When using this feature, you **HAVE** to use cloud provider credentials with a longer expiration than the cluster's life or static credentials. The autoscaler makes usage of this credentials

Author: ```suggestion
<p class="message--note"><strong>IMPORTANT: </strong> When using this feature, you <strong>HAVE</strong> to use cloud provider credentials with a longer expiration than the cluster's life or static credentials. The autoscaler makes usage of these credentials
```

> <p class="message--note"><strong>IMPORTANT: </strong> When using this feature, you <strong>HAVE</strong> to use cloud provider credentials with a longer expiration than the cluster's life or static credentials. The autoscaler makes usage of these credentials

Reviewer: ```suggestion
<p class="message--note"><strong>IMPORTANT: </strong> When using this feature, you <strong>must</strong> use cloud provider credentials with a longer expiration than the cluster's life or static credentials. The autoscaler makes usage of these credentials
```
